### ISSUE TITLE
Go 1.10: memory.go:227: Sprintf format %d has arg changeID of wrong type string

### ISSUE THREAD

**Issue by nim-nim** [theupdateframework/notary#1322]
a079b57c81eb71f4a754c29b963bf008bccfa3ef does not pass unit tests with Go 1.10. At least:

```
+ GOPATH=/builddir/build/BUILD/notary-a079b57c81eb71f4a754c29b963bf008bccfa3ef/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\'''
# github.com/theupdateframework/notary/server/storage
./memory.go:227: Sprintf format %d has arg changeID of wrong type string
./sqldb.go:284: Sprintf format %d has arg changeID of wrong type string

```

**Comment by justincormack** [comment]
fixed in #1332

**Comment by nim-nim** [comment]
Thanks a lot!
